### PR TITLE
Update translation strings as of 20180716T200628

### PR DESCRIPTION
**Pull request by folio-translations**

### CODE REVIEW

File: translations/stripes-components/en.json [hunk 1]
```diff
@@ -1,58 +1,51 @@
...
+    "showLessAddresses": "Show less",
+    "showMoreAddresses": "Show more addresses",
+    "showOrHideDatepicker": "Show\/hide datepicker",
+    "showOrHideTimepicker": "Show\/hide timepicker",
+    "stateProviceOrRegion": "State\/Prov.\/Region",
+    "submit": "Submit",
+    "tableColumnCount": "Table with {count} columns",
+    "validatingInProcess": "Validating...",
+    "zipOrPostalCode": "Zip\/Postal Code"
+}
```
Comment (JohnC-80) [inline translations/stripes-components/en.json]: Appears we're missing some of the translations. Looks like the ones for `selection.*` are absent from the new set. The difference in lines tipped me...

Comment (JohnC-80) [reply translations/stripes-components/en.json]: As well as `button.*`

File: translations/stripes-components/en.json [hunk 2]
```diff
@@ -1,58 +1,51 @@
...
+    "metaSection.source": "Source: {source}",
+    "primaryAddress": "Primary address",
+    "removeFields": "remove fields for {item} {num}",
+    "removeSelection": "Remove selection",
+    "saveChangesToThisItem": "Save changes to this item",
+    "screenReaderLabel": "{label} format is {screenReaderFormat} use arrow keys to navigate dates. Enter key to confirm. {screenReaderMessage}",
+    "selectDay": "Select {dayOfWeek}, {date}",
+    "showLessAddresses": "Show less",
+    "showMoreAddresses": "Show more addresses",
+    "showOrHideDatepicker": "Show\/hide datepicker",
```
Comment (zburke) [inline translations/stripes-components/en.json]: Why are the forward slashes escaped?

File: translations/stripes-components/en.json [hunk 3]
```diff
@@ -1,58 +1,51 @@
...
+    "showOrHideTimepicker": "Show\/hide timepicker",
+    "stateProviceOrRegion": "State\/Prov.\/Region",
```
Comment (zburke) [inline translations/stripes-components/en.json]: Typo in this key, but that's not new. Can we add the correct key, fix our apps, and then remove this?

File: translations/stripes-components/en.json (hunk 2 above)
Comment (dltj) [reply translations/stripes-components/en.json]: _sigh_: there is an export option for this:

> Unescape forward slashes: Turn on to skip escaping forward slashes.

The default is to escape forward slashes.  I've fixed the export API call parameters.

Comment (dltj) [reply translations/stripes-components/en.json]: Wow -- the description of the parameter in the API call is even more mentally twisting:

> Enable to leave forward slashes unescaped.

File: translations/stripes-components/en.json (hunk 1 above)
Comment (dltj) [reply translations/stripes-components/en.json]: Thanks, John -- I found that the automated notification to Lokalise on merge-to-master is _not_ automatically set up.  It needs to be set up for each GitHub repository.  I'll go through and do that.  In the meantime, I'm going to do a manual fetch from GitHub and create a new pull request on stripes-components.

**Comment by folio-translations** [comment]
New attempt at #499

**Pull request closed without merging**